blurring_as_a_service/performace_evaluation_pipeline/components/evaluate_with_cvt_metrics.py

```diff
@@ -26,13 +28,34 @@
     display_name="Evaluation with CVT metrics",
     environment=f"azureml:{aml_experiment_settings['env_name']}:{aml_experiment_settings['env_version']}",
     code="../../../",
+    is_deterministic=False,
 )
 def evaluate_with_cvt_metrics(
-    mounted_dataset: Input(type="uri_folder"), yolo_output_folder: Input(type="uri_folder")  # type: ignore # noqa: F821
+    mounted_dataset: Input(type="uri_folder"),  # type: ignore # noqa: F821
+    yolo_output_folder: Input(type="uri_folder"),  # type: ignore # noqa: F821
+    annotations_for_custom_metrics: Input(type="uri_file"),  # type: ignore # noqa: F821
+    experiment_name: str,  # type: ignore # noqa: F821
 ):
     true_path = f"{mounted_dataset}/labels/val"
-    pred_path = f"{yolo_output_folder}/exp/labels"
+    pred_path = f"{yolo_output_folder}/{experiment_name}/labels"
+    pred_path_tagged = f"{yolo_output_folder}/{experiment_name}/labels_tagged"
 
+    # ======== Total Blurred Area metric ========= #
     collect_and_store_tba_results_per_class_and_size(
         true_path, pred_path, markdown_output_path="tba_results.md"
     )
+
+    # ======== False Negative Rate metric ========= #
+
+    if pathlib.Path.exists(pathlib.Path(pred_path_tagged)):
```

Comment:
You can also from pathlib import Path and just use Path, probably it would look cleaner ;)